### DISCUSSION TITLE
Wierd scrollbar appears in autogrow text input on mobile

### DISCUSSION THREAD

**Discussion by yanliang2004** [quasarframework/quasar#15900] [General - Components / Directives / etc]
code:
`
<q-input
  label="content"
  filled
  autogrow
></q-input>
`
when opened in android browser, a scrollbar appears in the input. The scrollbar disappears when the input area grows.
<img width="180" alt="input" src="https://github.com/quasarframework/quasar/assets/33000802/d7d9213a-1462-4ad8-8e30-86b5c8f1b5b6">

I suggest that for autogrow inputs, the user can set an initial height to the input.

**Comment by pdanpdan** [comment]
Can you please provide a reproduction?

**Reply by yanliang2004** [reply]
Device: Redmi K40
OS: MIUI 13.0.7 Stable (Android version 12)

Android Browser: V17.6.280522
Edge Browser: 113.0.1774.38



`// package.json:

{
  "name": "q-test",
  "version": "0.0.1",
  "description": "quasar test",
  "productName": "Quasar App",
  "author": "alex <yanliang2004@qq.com>",
  "private": true,
  "scripts": {
    "lint": "eslint --ext .js,.vue ./",
    "format": "prettier --write \"**/*.{js,vue,scss,html,md,json}\" --ignore-path .gitignore",
    "test": "echo \"No test specified\" && exit 0"
  },
  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "core-js": "^3.6.5",
    "quasar": "^2.0.0",
    "vue": "^3.0.0",
    "vue-router": "^4.0.0",
    "vuex": "^4.0.1"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.13.14",
    "@quasar/app": "^3.0.0",
    "eslint": "^7.14.0",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-vue": "^7.0.0",
    "eslint-webpack-plugin": "^2.4.0",
    "prettier": "^2.5.1"
  },
  "browserslist": [
    "last 10 Chrome versions",
    "last 10 Firefox versions",
    "last 4 Edge versions",
    "last 7 Safari versions",
    "last 8 Android versions",
    "last 8 ChromeAndroid versions",
    "last 8 FirefoxAndroid versions",
    "last 10 iOS versions",
    "last 5 Opera versions",
    "> 1%"
  ],
  "engines": {
    "node": ">= 12.22.1",
    "npm": ">= 6.13.4",
    "yarn": ">= 1.21.1"
  }
}
`
